src/ethereUtils.ts

```
@@ -0,0 +1,1001 @@
+        const result: Uint8Array[] = [];
+        for (const v of value) result.push(_pack(baseType, v, true));
+        return getBytes(concat(result));
+    }
+
+    assertArgument(false, "invalid type", "type", type);
+}
+
+export function solidityPacked(types: ReadonlyArray<string>, values: ReadonlyArray<unknown>): Hex {
+    assertArgument(types.length === values.length, "wrong number of values", "values", values);
+    const tight: Uint8Array[] = [];
+    types.forEach((type, index) => tight.push(_pack(type, values[index])));
```

Biome (2.4.15)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)